BraintreeCore/src/main/java/com/braintreepayments/api/BraintreeClient.kt

```diff
@@ -429,7 +429,15 @@ open class BraintreeClient @VisibleForTesting internal constructor(
      */
     @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
     fun reportCrash() = authorizationLoader.authorizationFromCache?.let { authorization ->
-        analyticsClient.reportCrash(applicationContext, sessionId, integrationType, authorization)
+        getConfiguration { configuration, _ ->
+            analyticsClient.reportCrash(
+                applicationContext,
+                configuration ?: null,
```

Comment:
```suggestion
                configuration,
```
If config is null it'll just pass as null 
```suggestion
                configuration,
```